.github/workflows/deployment.yml

```diff
@@ -0,0 +1,27 @@
+name: Deployment
+on: 
+  push:
+jobs:
+  Deployment:
+    runs-on: ubuntu-latest
+    steps:
+    - id: files
+      uses: jitterbit/get-changed-files@v1
+      continue-on-error: true
+
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+
+    - name: Setup Python
+      uses: actions/setup-python@v2.2.2
+      with:
+        python-version: 3.8
```

Comment:
why not use NodeJS for the deployment script, which reduces a dependency and keeps everything the same language